network/dns-cloud/how-to/delete-domain.mdx

```diff
@@ -27,7 +27,7 @@ dates:
     <Lightbox src="scaleway-dns-delete-domain.webp" alt="" />
     <Message type="important">
 
-    This will permanently delete your external domain and its related zones from Scaleway DNS. This action is irreversible and your domain name may become unresolvable. 
+    This will permanently delete your external domain and its related zones from Scaleway Domains & DNS. This action is irreversible and your domain name may become unresolvable. 
```

Comment:
```suggestion
    This will permanently delete your external domain and its related zones from Scaleway Domains and DNS. This action is irreversible and your domain name may become unresolvable. 
```